### DISCUSSION TITLE
Configure MeshCentral to use Let's Encrypt with Cloudflare API to obtain SSL certificate over port 443.

### DISCUSSION THREAD

**Discussion by adimate71** [Ylianst/MeshCentral#5023] [Q&A]
Hi,

Please let me know if it is possible to generate the domain certificate with Letsencrypt by using the Cloudflare API. I would like to avoid opening port 80.

Thank you

**Comment by adimate71** [comment]
Can you please help?

**Reply by silversword411** [reply]
certbot, with api key and configuration is needed.

Paid support is available if you can't figure it out: https://h.ineedhelpwithtech.com/
